server/src/main/java/org/elasticsearch/index/mapper/FieldTypeLookup.java

```diff
@@ -205,6 +220,15 @@ Set<String> sourcePaths(String field) {
         return fieldToCopiedFields.containsKey(resolvedField) ? fieldToCopiedFields.get(resolvedField) : Set.of(resolvedField);
     }
 
+    Set<String> getFieldsForModel(String model) {
+        Set<String> fields = fieldsForModels.get(model);
+        return fields != null ? fields : Collections.emptySet();
```

Comment:
Any preference towards using `Set.of()` vs `Collections.emptySet()` to generate an empty set?